android/sdk/src/main/java/com/taobao/weex/utils/ImageDrawable.java

```diff
@@ -132,6 +132,9 @@ protected void onDraw(Shape shape, Canvas canvas, Paint paint) {
     if (Build.VERSION.SDK_INT == Build.VERSION_CODES.LOLLIPOP) {
       // fix api 21 PaintDrawable crash
       paint.setAntiAlias(false);
+    } else {
+      // make android support anti-aliased image
+      paint.setFlags(Paint.ANTI_ALIAS_FLAG|Paint.FILTER_BITMAP_FLAG);
```

Comment:
paint.setAntiAlias(ture) will be enough, I think

Reply:
```paint.setAntiAlias(ture)``` is not enough.
I tested it on Android 8.0. Only with this line of code, the image doesn't appear jagged.

Reply:
Please add `imageDrawable.getPaint().setFilterBitmap(true);` when creating `ImageDrawable`, no need to invoke setFlags during `draw()`